src/reanimated2/mock.js

```diff
@@ -25,6 +25,9 @@ const ReanimatedV2 = {
     cb && setTimeout(() => cb(true), 0);
     return 0;
   },
+  withDelay: (_, animationValue) => {
+    return animationValue;
+  },
   cancelAnimation: NOOP,
   delay: (_, b) => b,
```

Comment:
Could you also remove this one? It's an old name for `withDelay`, your implementation should replace it.
Same with sequence and repeat, but I think the implementation of those should be changed as well, so we can do it in the follow-up (but you can fix it here if you want to).